### PR TITLE
[script.video.bgtime_tv] v1.0.1

### PR DESCRIPTION
**Pull request by portokala123**
### Description
This add-on is for your service https://bgtime.tv/. It enables our subscribers to watch bulgarian tv programs on Kodi.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by romanvm** [comment]
LGTM, other than code style

File: script.video.bgtime_tv/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Auto detect text files and perform LF normalization
+* text=auto
```
Comment (romanvm) [inline script.video.bgtime_tv/.gitattributes]: Please remove Git service files

File: script.video.bgtime_tv/.gitignore [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+# virtualenv
+.venv/
+venv/
+ENV/
+
+# Spyder project settings
+.spyderproject
+
+# Rope project settings
+.ropeproject
```
Comment (romanvm) [inline script.video.bgtime_tv/.gitignore]: The same

File: script.video.bgtime_tv/addon.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.video.bgtime_tv" name="BGTime.TV" version="1.0.1" provider-name="mariapmarkova@gmail.com">
+	<requires>
+		<import addon="xbmc.python" version="2.24.0"/>
```
Comment (romanvm) [inline script.video.bgtime_tv/addon.xml]: It's v.2.25.0 for Krypton

**Comment by romanvm** [comment]
Please rebase 2 commits into one

**Comment by enen92** [comment]
ping @portokala123

**Pull request closed without merging**